bundles/automation/org.eclipse.smarthome.automation.rest/src/main/java/org/eclipse/smarthome/automation/rest/internal/RuleResource.java

```diff
@@ -97,16 +97,22 @@ private EnrichedRuleDTO enrich(Rule rule) {
     @Consumes(MediaType.APPLICATION_JSON)
     @ApiOperation(value = "Creates a rule.")
     @ApiResponses(value = { @ApiResponse(code = 201, message = "Created"),
-            @ApiResponse(code = 409, message = "Creation of the rule is refused. Rule with the same UID already exists.") })
+            @ApiResponse(code = 409, message = "Creation of the rule is refused. Rule with the same UID already exists."),
+            @ApiResponse(code = 400, message = "Creation of the rule is refused. Missing required parameter.") })
```

Comment:
Jersey does not do any validation. It just try to fill RuleResource from passed JSON object. If the JSON is wrong and it does not contain any of the RuleResource properties it just doesn't fill them.
I've created rest request with  wrong JSON structure (I've change the trigger's configuration to be defined in "config" node instead of "configuration" node) and the result was a response 201 with created rule which has empty triggers configuration.